core/src/main/scala/kafka/server/streamaspect/ElasticReplicaManager.scala

```diff
@@ -967,6 +972,8 @@ class ElasticReplicaManager(
             }
           }
         } finally {
+          tracker.release()
+          tracker.closed()
```

Comment:
`release` is already called in `closed`:
>        public synchronized void closed() {
>            transition(Status.CLOSED);
>            release();
>        }

Reply:
Need double release to remove the tracker